in_toto/runlib.py

```diff
@@ -251,9 +251,12 @@ def record_artifacts_as_dict(artifacts, exclude_patterns=None,
         # Apply exlcude patterns on the normalized file paths returned by walk
         if exclude_patterns:
           filepaths = _apply_exclude_patterns(filepaths, exclude_patterns)
+        # FIXME: this is necessary to provide consisency between windows filepaths and
+        # *nix filepaths. A better solution may be in order though...
```

Comment:
I am not convinced that this PR would fix all issues on Windows.

For example, are there checks to prevent absolute paths such as `C://Program Files/in-toto/run.py`?